Production/python/Run2017B-PromptReco-v2/HTMHT_cff.py

```diff
@@ -0,0 +1,44 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I was finding that the updated JSON does have runs in v2 of the 2017RunB dataset